src/assets/icons/close-icon.svg

```diff
@@ -1,4 +1,3 @@
-<svg width="24" height="24" viewBox="0 0 24 24" fill="none"
-  xmlns="http://www.w3.org/2000/svg">
-  <path fill-rule="evenodd" clip-rule="evenodd" d="M8.04228 18.3176L12.2497 14.1094L16.4575 18.3173C16.7009 18.5606 17.0958 18.5606 17.3395 18.3173L18.3173 17.3395C18.5606 17.0961 18.5606 16.7012 18.3173 16.4575L14.109 12.2501L18.3169 8.04228C18.5602 7.7989 18.5602 7.40401 18.3169 7.16025L17.3391 6.1825C17.0958 5.93917 16.7009 5.93917 16.4571 6.1825L12.2497 10.3907L8.0419 6.18289C7.79852 5.93955 7.40362 5.93955 7.15987 6.18289L6.1825 7.16025C5.93917 7.40359 5.93917 7.79852 6.1825 8.04228L10.3903 12.2501L6.1825 16.4579C5.93917 16.7012 5.93917 17.0961 6.1825 17.3399L7.16025 18.3176C7.40362 18.561 7.79852 18.561 8.04228 18.3176Z" fill="currentColor"/>
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M18.7 5.3C18.3 4.9 17.7 4.9 17.3 5.3L12 10.6L6.7 5.3C6.3 4.9 5.7 4.9 5.3 5.3C4.9 5.7 4.9 6.3 5.3 6.7L10.6 12L5.3 17.3C4.9 17.7 4.9 18.3 5.3 18.7C5.5 18.9 5.7 19 6 19C6.3 19 6.5 18.9 6.7 18.7L12 13.4L17.3 18.7C17.5 18.9 17.8 19 18 19C18.2 19 18.5 18.9 18.7 18.7C19.1 18.3 19.1 17.7 18.7 17.3L13.4 12L18.7 6.7C19.1 6.3 19.1 5.7 18.7 5.3Z" fill="#65647E"/>
```

Comment:
This is gonna break the webapp because no "currentColor" fill. Why were they replaced?

Reply:
Yes, i changed it back. This process should be automized in future.